scripts/ci/run_perftest.sh

```diff
@@ -315,35 +255,100 @@ org_flag() {
   esac
 }
 
-# Generate a DBRP mapping for use by InfluxQL queries.
+create_dbrp() {
 curl -XPOST -H "Authorization: Token ${TEST_TOKEN}" \
   -d "{\"org\":\"${TEST_ORG}\",\"bucketID\":\"$(bucket_id)\",\"database\":\"$db_name\",\"retention_policy\":\"autogen\"}" \
   http://${NGINX_HOST}:8086/api/v2/dbrps
+}
+
+##########################
+## Run and record tests ##
+##########################
+
+# Generate queries to test.
+query_files=""
+for usecase in window-agg group-agg bare-agg group-window-transpose-low-card group-window-transpose-high-card iot metaquery multi-measurement; do
```

Comment:
Not a blocker, but is it possible to group these usecases into environment variables? They are thrown around a lot in the case statements and helper functions, and it's a  bit hard to keep them straight.

Reply:
Hrmm I definitely see what you mean from a maintainability standpoint, and it's fairly difficult to add or change the queries/usecases that run now that we've got quite a few of them. I have some ideas on how to improve this but it might require some more significant refactoring for what I've got pictured so I'll save that for a later PR.